CSharp/BatchExplorer/Plugins/LegacyAccountSupport/LegacyAccount.cs

```diff
@@ -17,6 +17,8 @@ public class LegacyAccount
         public string BatchServiceUrl { get; set; }
         public byte[] SecureKey { get; set; }
         public Guid UniqueIdentifier { get; set; }
+        public string LinkedStorageAccountName { get; set; }
```

Comment:
I was blindly changing this everywhere. I removed Linked Storage from LegacyAccount